src/components/Footer/index.tsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import { Text } from '@deriv/ui';
-import styles from './Footer.module.scss';
-import Translate from '@docusaurus/Translate';
 import { LabelPairedArrowUpRightSmRegularIcon } from '@deriv/quill-icons';
 import { Button } from '@deriv/ui';
 import { SocialTelegramBlackIcon } from '@deriv/quill-icons';
 import { LabelPairedEnvelopeCaptionBoldIcon } from '@deriv/quill-icons';
+import Translate from '@docusaurus/Translate';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import CustomAccordion from '../CustomAccordion';
+import styles from './Footer.module.scss';
 
 const Footer = () => {
+  const {
+    i18n: { currentLocale },
+  } = useDocusaurusContext();
+
+  function generateExternalURL(path = '') {
+    const isLocaleEn = currentLocale === 'en';
+    return `https://deriv.com${!isLocaleEn ? `/${currentLocale}` : ''}/${path}`;
+  }
```

Comment:
```suggestion
  const generateExternalURL = (path = '') => useMemo({
    const is_en = currentLocale === 'en';
    return `https://deriv.com${!is_en ? `/${currentLocale}` : ''}/${path}`;
  }, [currentLocale, path])
```

@ahmed-deriv can we do this?